### PR TITLE
Start depending on the upstream out-of-tree Wormhole

### PR DESCRIPTION
**Pull request by drozdziak1**
When finished, this will let us drop the in-tree copy of bridge and solitaire sources in favor of verbatim upstream sources downloaded from `certusone/wormhole`

### CODE REVIEW

File: devnet/node.yaml [hunk 1]
```diff
@@ -89,18 +87,40 @@ spec:
...
+            # - http://algorand:4001
+            # - --algorandAlgodToken
+            # - "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
             - --solanaContract
             - Bridge1p5gheXUvJ6jGWGeCsgPKgnE3YgdGKRVCMY9o
-            - --solanaWS
-            - ws://solana-devnet:8900
-            - --solanaRPC
-            - http://solana-devnet:8899
+            # - --solanaWS
```
Comment (ali-bahjati) [inline devnet/node.yaml:120]: Why are these commented? 🤔

File: solana/pyth2wormhole/program/src/migrate.rs [hunk 2]
```diff
@@ -64,6 +59,18 @@ pub fn migrate(ctx: &ExecutionContext, accs: &mut Migrate, data: ()) -> SoliResu
         ));
     }
 
+    if *accs.system_program.key != system_program::id() {
```
Comment (ali-bahjati) [inline solana/pyth2wormhole/program/src/migrate.rs:62]: Where is it used? by `create` method below?

File: devnet/node.yaml (hunk 1 above)
Comment (drozdziak1) [reply devnet/node.yaml:120]: I copied most of this file verbatim from upstream, I figured that if I only keep the changes minimal, it should make catching it up to wormhole easier in the future. Not feeling strongly about this, just looking out for future us

File: solana/pyth2wormhole/program/src/migrate.rs (hunk 2 above)
Comment (drozdziak1) [reply solana/pyth2wormhole/program/src/migrate.rs:62]: correct, we were using a solitaire-client thingy for this previously, which would include the system program automatically for accounts that will possibly need it for creation. Latest solitaire is on-chain focused.

**Pull request merged**